### PR TITLE
Document how to use the presenter console.

### PR DESCRIPTION
**Pull request by mkohler**

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -78,6 +78,25 @@ Optional arguments:
...
         Show program's version number and exit
 
+Presenter Console
+-----------------
+
+The presenter console feature is designed for showing an annotated version of
+the presentation on the local laptop display, while showing the presentation
+itself on a projector.
+
+To use this feature, open the presentation in a browser, and press "P". A new
```
Comment (yeyeto2788) [inline docs/usage.rst]: I might be a bit picky but do you mind changing the "P" for 
```text
`p`
```

File: docs/usage.rst [hunk 2]
```diff
@@ -96,7 +96,7 @@ Move to New Window. With Chromium, drag the tab title down and the tab will be
 moved into a new browser window.
 
 Finally, drag the presentation window onto the projector's part of the
-desktop.
+desktop, and press F11 to make the presentation fullscreen.
```
Comment (yeyeto2788) [inline docs/usage.rst]: Change the F11 to 
```text
`F1`
```

And sorry for being so picky.

Comment (mkohler) [reply docs/usage.rst]: Done.

File: docs/usage.rst (hunk 1 above)
Comment (mkohler) [reply docs/usage.rst]: Done.

**Pull request merged**